res/database/Default/user/therob.xml

```diff
@@ -234,10 +262,12 @@
 			<title>
 				<de>Sonstiges - das Thema</de>
 				<en>Various - the topic</en>
+				<pl>Różne - tematy</pl>
 			</title>
 			<description>
 				<de>[0|Full] hat [1|Full], [2|Full], [3|Full] und [4|Full] eingeladen. Das Ziel ein intensives Gespräch über Sonstiges. Wie wird die Quote? Was gehört da rein? Wie wirkt es sich aus? Sehr spannende Expertenrunde. Vor allem, als der Streit ausbricht.</de>
 				<en>[0|Full] has invited [1|Full], [2|Full], [3|Full] and [4|Full]. The goal an intensive conversation about 'Various'. What goes into it? What is the impact? What will be the viewing rates? Very exciting round of experts. Especially when the argument breaks out.</en>
+				<pl>[0|Full] zaprasza gości. To: [1|Full], [2|Full], [3|Full] i [4|Full]. Celem jest intensywna rozmowa na "Różne" tematy . Co się na to składa? Jaki jest wpływ? Jakie będą wskaźniki oglądalności? Bardzo ekscytująca runda ekspertów. Zwłaszcza gdy wybuchnie kłótnia.</pl>
```

Comment:
Genre 'Various' was recently changed to 'Sundry' in the English version, the English version will be updated here also.

So these (and other similar entries in this file) make fun of the 'Sundry' genre in TVTower. I don't know about Polish grammar but maybe it would be better to also use "Różnorodne" (already in quotes anyway) to make that clearer to the player?

Reply:
ok. adapted